docs/csharp/programming-guide/strings/index.md

```diff
@@ -64,11 +64,22 @@ A string is an object of type <xref:System.String> whose value is text. Internal
 >  At compile time, verbatim strings are converted to ordinary strings with all the same escape sequences. Therefore, if you view a verbatim string in the debugger watch window, you will see the escape characters that were added by the compiler, not the verbatim version from your source code. For example, the verbatim string @"C:\files.txt" will appear in the watch window as "C:\\\files.txt".  
   
 ## Format Strings  
- A format string is a string whose contents can be determined dynamically at runtime. You create a format string by using the static <xref:System.String.Format%2A> method and embedding placeholders in braces that will be replaced by other values at runtime. The following example uses a format string to output the result of each iteration of a loop:  
+ A format string is a string whose contents are determined dynamically at runtime. Format strings are created by embedding *interpolated expressions* or placeholders inside of braces within a string. Everything inside the braces (`{...}`) will be resolved to a value and output as a formatted string at runtime. There are two methods used to create interpolated strings:
+
+### String Interpolation  
+Available in C# 6.0 and later, [*interpolated strings*](../../../csharp/language-reference/tokens/interpolated) are identified by the `$` special character and include interpolated expressions in braces. If you are new to string interpolation, see the [String interpolation - C# interactive tutorial](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/interpolated-strings) for a quick overview.
+
+Utilizing string interpolation improves the syntax and readability of C# code. String interpolation achieves the same results as the `String.Format` method below, but represents an improvement due to ease of use and inline clarity.
+
+`Sample...`
+
+### Format Method
+The [format method](https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=netframework-4.7.2) utilizes placeholders in braces to create a format string. The following example uses a format string to output the result of each iteration of a loop:
```

Comment:
Instead of "[format method](https://...)" it would be better to use the xref syntax:  `<xref:System.String.Format%2A?displayProperty=nameWithType>`

Reply:
I noted your xref has a displayProperty and all the others on the page end at 2A. Just curious what specific effect this will have?

Reply:
the "displayProperty=nameWithType" means the type and the member are displayed. 

All the keys should be in the [Contributor Guide](https://docs.microsoft.com/contribute/dotnet-style-guide#links-to-apis)